cmd/es-utils/utils/utils.go

```diff
@@ -371,10 +401,23 @@ func UploadBlobs(
 		log.Info("Timed out for receipt, query contract for data hash...")
 	}
 	// if wait receipt timed out or failed, query contract for data hash
-	return getKvInfo(pc, contractAddr, len(blobs))
+	return getKvInfo(pc, len(blobs))
```

Comment:
split to another PR?

Reply:
What for?

Reply:
`queryBlobBaseFee` is used by mining tests.